docs/proposals/20240819-build-iot-system-configuration-isolation-on-nodepool.md

```diff
@@ -0,0 +1,154 @@
+  - [Table of Contents](#table-of-contents)
+  - [Summary](#summary)
+  - [Motivation](#motivation)
+    - [Goals](#goals)
+  - [Proposal](#proposal)
+    - [User Stories](#user-stories)
+    - [Implementation Details](#implementation-details)
+    - [Test Plan](#test-plan)
+  - [Implementation History](#implementation-history)
+
+
+## Summary
+
+Openyurt gave users the ability to customize iot systems, but it's currently not isolated enough for nodepool.
+This proposal aims to provide multiple PlatformAdmin deployments within the same namespace, and to allow users the ability to customize the configuration of a nodepool.
+
+## Motivation
+
+Suppose now you need to expand several nodepools with the same configuration, the current plan is to create several new Platformadmins with the same configuration.Obviously, the operability and reusability of this solution is poor.
+One potential enhancement involves modifying the mapping between Platformadmin and nodepools to a one-to-many relationship, that is, changing the poolName in PlatformadminSpec to pools to correspond to multiple nodepools.
+In this proposal, users can deploy multiple node pools with the same configuration by creating a single PlatformAdmin.
+
+
+### Goals
+
+- Provide multiple PlatformAdmin deployments within the same namespace
+
+- Allow users the ability to customize the configuration of a nodepool
+
+- Add unit test for platform_ admin_controller and modify corresponding e2e test
+
+
+## Proposal
+
+### User Stories
+
+- As a user,I wanted to customize configurations based on the nodepool dimension, thereby achieving the reuse of both custom configurations and Platformadmin.
+
+### Implementation Details
+
+PlatformAdmin has evolved from the previous version of the EdgeX CRD and serves as an abstraction for the edge device management platform. Users simply input the platform settings, the name of the NodePool to be deployed, the version to be deployed, and so on, to deploy a complete edge device management platform within the node pool. 
+
+The platformadmin-controller, integrated within yurt-manager, is responsible for parsing the PlatformAdmin CR into the corresponding configmap, service, and yurtappset, thereby realizing the deployment of the edge device management platform. 
+
+Users have the capability to customize a PlatformAdminFramework that is initialized with a standard configuration, followed by the creation of a PlatformAdmin. After this step, the platformadmin-controller will automatically initiate the reconciliation process to handle services and YurtAppSets.
+
+However, a notable drawback of this process is that, even when multiple nodepools share identical configurations, it necessitates the creation of multiple PlatformAdmins.  This redundancy can lead to unnecessary administrative overhead and complexity.
+
+![platformadmin-old-frame](../img/20240819-build-iot-system-configuration-isolation-on-nodepool/platformadmin-old-frame.png)
```

Comment:
Auto-Controller --> Auto-Collector

Reply:
Now, the image content has been updated.